subsys/net/lib/sockets/Kconfig

```diff
@@ -96,6 +96,12 @@ config NET_SOCKETS_OFFLOAD
 	  See NET_OFFLOAD for a more deeply integrated approach which offloads
 	  from the net_context() API within the Zephyr IP stack.
 
+config NET_SOCKETS_RAW
+	bool "Enable raw socket support"
+	default n
+	help
+	  The value depends on your network needs.
```

Comment:
So, here will go description of what flavors of RAW sockets we actually support.

Reply:
Added more text in new version.